### PR TITLE
Play Pause

### LINKED ISSUE #19: Play Button
Add a button to start automatically stepping through the time points at an adjustable rate.

### PR DESCRIPTION
**Pull request by alexanderkyim**
---
name: Play Pause Button
about: Added a play and pause button to step through the different frames of the 

---

## Issues Addressed
Closes #19 

## What has changed
Include each file that was changed, how it was changed (in broad terms), and why it was changed.
Example:
    tesla-ver/bubble_chart.py:
          play_pause_switch(n_clicks) sets the state of an interval component.
          play_increment(n_intervals, time_value) updates the time value, which fires callbacks to update the graph.

## Checklist of pre-requisites
All of these need to be checked in order to request a review on the Pull Request
-   [X] Does the code run?  
-   [X] Does the code follow the repository style?  
-   [X] Is the code tested?  
-   [X] Has Black been run on all the python files in the repository (with line length 120 `black -l 120 *`)?

## Additional notes

# Enhancement

## How to use the feature
Upload data as per normal, and then after selecting all the dropdown values, press the play/pause button that steps through the frames on a time interval

## If the user interface was modified include screenshots here
Screen Shot 2020-07-15 at 13 03 10

### CODE REVIEW

File: tesla_ver/bubble_chart.py [hunk 1]
```diff
@@ -18,19 +20,25 @@ def generateBubbleChart(server):
...
         component."""
 
         df = None
 
-        if None in [list_of_contents, list_of_filenames]:
+        if None in [list_of_contents, list_of_filenames] or len(list_of_contents) != 1:
             raise PreventUpdate
 
-        def parse_contents(contents, filenames):
+        def upload_string_to_df(content):
```
Comment (DSWallach) [inline tesla_ver/bubble_chart.py:35]: Add docstring

File: tesla_ver/bubble_chart.py [hunk 2]
```diff
@@ -162,31 +183,24 @@ def update_figure(
...
-                Scattergl(
-                    x=df_by_value[x_column_name],
-                    y=df_by_value[y_column_name],
-                    mode="markers",
-                    opacity=0.9,
-                    name=entity,
-                    hovertext=df_by_value[annotation_column_name].values.tolist(),
-                ),
-            )
+        df_by_time = pd.DataFrame.from_dict(ast.literal_eval(json.loads(json_data).get(str(time_value))))
```
Comment (DSWallach) [inline tesla_ver/bubble_chart.py:186]: Add a comment on the operations being performed here

Comment (alexanderkyim) [reply tesla_ver/bubble_chart.py:196]: addressed in 6e8fe20

File: tesla_ver/bubble_chart.py (hunk 1 above)
Comment (alexanderkyim) [reply tesla_ver/bubble_chart.py:35]: addressed in 6e8fe20

**Pull request merged**